fe/be-java-extensions/hudi-scanner/pom.xml

```diff
@@ -78,15 +78,16 @@ under the License.
             </exclusions>
         </dependency>
         <dependency>
```

Comment:
Why change the hadoop dependencies.

Reply:
Because there is a conflict between hudi and avro dependencies